tutorials/flaml-tutorial-aaai-23.md

```diff
@@ -0,0 +1,67 @@
+# AAAI 2023 Lab Forum - LSHP2: Automated Machine Learning & Tuning with FLAML
+
+## Session Information
+
+**Date and Time**: February 8, 2023 at 2-6pm ET.
+
+Location: Walter E. Washington Convention Center, Washington DC, USA
+
+Duration: 4 hours (3.5 hours + 0.5 hour break)
+
+For the most up-to-date information, see the [AAAI'23 Program Agenda](https://aaai.org/Conferences/AAAI-23/aaai23tutorials/)
+
+## [Lab Forum Slides](https://1drv.ms/b/s!Ao3suATqM7n7iokCQbF7jUUYwOqGqQ?e=cMnilV)
+
+## What Will You Learn?
+
+- What FLAML is and how to use FLAML to
+    - find accurate ML models with low computational resources for common ML tasks
+    - tune hyperparameters generically
+- How to leverage the flexible and rich customization choices
+  - finish the last mile for deployment
+  - create new applications
+- Code examples, demos, use cases
+- Research & development opportunities
+
+## Session Agenda
+
+### **Part 1. Overview of FLAML**
+
+- Overview of AutoML and FLAML
+- Basic usages of FLAML
+    - Task-oriented AutoML
+        - [Documentation](https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML)
+        - [Notebook: A classification task with AutoML](https://github.com/microsoft/FLAML/blob/tutorial-aaai23/notebook/automl_classification.ipynb); [Open In Colab](https://colab.research.google.com/github/microsoft/FLAML/blob/tutorial-aaai23/notebook/automl_classification.ipynb)
```

Comment:
Maybe we host the notebooks for each tutorial in a separate subfolder under the `tutorials` folder and update the links. Thus we can have everything in the `main` branch.